asm/Kptfile

```diff
@@ -93,6 +93,11 @@ openAPI:
         setter:
           name: anthos.servicemesh.tag
           value: 1.6.8-asm.9
+    io.k8s.cli.setters.anthos.servicemesh.canonicalServiceHub:
+      x-k8s-cli:
+        setter:
+          name: anthos.servicemesh.canonicalServiceHub
+          value: gcr.io/gke-release/asm/canonical-service-controller:1.6.8-asm.9          
```

Comment:
same version tag question - do we need to update?

Reply:
Responded above. same answer here.